src/style.css

```diff
@@ -53,6 +67,36 @@ header a {
   text-align: center;
 }
 
+#publish-comment-button {
+  margin-top: 20px;
+  padding: 1.3em 3em;
+  width: min-content;
+  text-align: center;
+  font-size: 16px;
+  text-transform: uppercase;
+  letter-spacing: 1.5px;
+  font-weight: 500;
+  color: #000;
+  background-color: #fff;
+  border: none;
+  border-radius: 45px;
+  box-shadow: 0 8px 15px rgba(0, 0, 0, 0.1);
```

Comment:
I think it would be better to stick with one measurement unit to keep our styles consistent e.g. just use `rem` for measurements instead of `px`